discovery-infra/tests/base_test.py

```diff
@@ -68,21 +70,10 @@ def cluster(self, api_client, request, nodes):
         clusters = []
 
         @JunitTestCase()
```

Comment:
why it is test case?

Reply:
You missed that PR? :)

Reply:
We can remove it if you think it shouldn't be there, But this is for another PR.